src/views/documents/SearchStore.ts

```diff
@@ -146,6 +146,23 @@ export class SearchV2Store {
     this.search(100, resetPagination);
   };
 
+  /**
+   * Add journalName to the current search, and remove any other in: tokens.
+   * i.e. search documents in one folder
+   */
+  searchJournal = (journalName: string) => {
+    const tokens = this._tokens.filter((t) => t.type !== "in");
```

Comment:
Ah, this should actually behave more like `setTokens`. If you:

* have an active multi-part search
* click sidebar and select a new journal

It should wipe the whole search, leaving only `in:`.

Reply:
Opted for a `setSearch` method, although perhaps simply `replace` is better (i.e. `searchStore.replace`).